services/openshift/openshift_option

```diff
@@ -3,3 +3,6 @@
 # Modify these options if you want to change openshift hostname
 OPENSHIFT_SUBDOMAIN="$(hostname).$(/opt/adb/openshift/get_ip_address).xip.io"
 IMAGE="docker.io/openshift/origin:latest"
+NO_PROXY=.xip.io,172.30.0.0/16
```

Comment:
This should also include the pod network (`10.1.0.0/16` or similar) e.g. for cases where the master reaches pods directly, like jmx console.
Also: if you run the vm in an environment where there is already some mandatory proxy setup (i.e. you already have some NO_PROXY values configured as required by your network) they should be added here too...